plugins/mcreator-localization/help/fr_FR/block/jump_factor.md

```diff
@@ -0,0 +1,4 @@
+Ce paramètre contrôle la hauteur des sauts des entités sur ce bloc.

```

Comment:
So, wouldn't you add "plante" too? Because it's for plants and blocks?

Reply:
A plant is a block, so...

Reply:
Yeah, I don't think it's necessary to add "plants". Other shared properties like hardness, resistance and luminance also refer only to "blocks"